tower-grpc-interop/travis-interop.sh

```diff
@@ -24,4 +24,6 @@ trap 'echo ":; killing test server"; kill ${SERVER_PID};' EXIT
 
 # run the interop test client against the server.
 cargo run -p tower-grpc-interop --bin client -- \
-    --test_case=empty_unary     # this is the only working test case...
\ No newline at end of file
+    # Add comma separated test cases
```

Comment:
i _think_ bash will see this comment as "whitespace" after the continuation `\`, i had problems with that when i was writing the script

Reply:
(just moving the comment to before the line starting with `cargo run ...` is probably the easiest solution)